src/Components/Actions/ActionsButton.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './ActionsButton.css';
```

Comment:
BTW, it will save all the CSS imports, in SCSS we do it centralized.

Reply:
Could you please elaborate?